syntaxes/ballerina.YAML-tmLanguage

```diff
@@ -630,11 +631,37 @@ repository:
         '0': { name: meta.arrow.ballerina storage.type.function.arrow.ballerina }
       end: (?=\;)|(?=\,)|(?=)(?=\);)
       patterns:
+      - include: '#naturalFunction'
       - include: '#statements'
       - include: '#punctuation-comma'
     - name: keyword.generator.asterisk.ballerina
       match: \*
 
+  naturalFunction:
```

Comment:
Isn't this `naturalExpression`?

Just to clarify, a natural function is just an expression-bodied function where the expression is a natural expression. We've just introduced a specific term for such functions. 

```ballerina
function getRandomNumber(int startN, int endN) returns int|error 
   => natural { Give me a random number between ${startN} and ${endN} };
```

So a natural expression is just another kind of an expression that can be used anywhere.

```ballerina
int x = check natural { Give me a random number between ${startN} and ${endN} }
```

Natural function is a special kind of function similar to data mapper functions.